MissingResourceException in TaxonomyTemplate

**Describe the bug**
When using the `name.abuchen.portfolio.jar` in my own program (to modify the portfolio.xml file) I go a MissingResource-Exception when accessing `TaxonomyTemplate`. This is due to the absolut path in "ResourceBundle.getBundle("/META-INF/taxonomy...". This works fine within an OSGI-Bundle but doesn't work in pure Java application. 

**To Reproduce**
```
public static void main(String[] args) throws IOException {
   System.out.println(TaxonomyTemplate.list());
}
```
```
Exception in thread "main" java.lang.ExceptionInInitializerError
	at de.tomsplayground.peanuts.portfolio.PortfolioExport.main(PortfolioExport.java:52)
Caused by: java.util.MissingResourceException: Can't find bundle for base name /META-INF/taxonomy/assetclasses, locale de_DE
	at java.base/java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:2055)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1689)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1593)
	at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1556)
	at java.base/java.util.ResourceBundle.getBundle(ResourceBundle.java:857)
	at name.abuchen.portfolio.model.TaxonomyTemplate.<init>(TaxonomyTemplate.java:35)
	at name.abuchen.portfolio.model.TaxonomyTemplate.<clinit>(TaxonomyTemplate.java:19)
	... 1 more
```

**Fix**

When using a relativ path like this "getBundle("META..." this works in both situations, with and without OSGI context.

--- a/name.abuchen.portfolio/src/name/abuchen/portfolio/model/TaxonomyTemplate.java
+++ b/name.abuchen.portfolio/src/name/abuchen/portfolio/model/TaxonomyTemplate.java
@@ -32,7 +32,7 @@ public final class TaxonomyTemplate
     {
         this.id = id;
 
-        ResourceBundle bundle = ResourceBundle.getBundle("/META-INF/taxonomy/" + id); //$NON-NLS-1$
+        ResourceBundle bundle = ResourceBundle.getBundle("META-INF/taxonomy/" + id); //$NON-NLS-1$
         this.name = getString(bundle, "name"); //$NON-NLS-1$
     }
 
@@ -99,7 +99,7 @@ public final class TaxonomyTemplate
      */
     /* package */Taxonomy buildFromTemplate()
     {
-        ResourceBundle bundle = ResourceBundle.getBundle("/META-INF/taxonomy/" + id); //$NON-NLS-1$
+        ResourceBundle bundle = ResourceBundle.getBundle("META-INF/taxonomy/" + id); //$NON-NLS-1$
 
         Taxonomy taxonomy = new Taxonomy(id, name);
 

